### PR TITLE
town update

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request
unique towner variants of pilgrim classes

most of them have weaker physical stats and/or fewer known skills
but they tend to have higher intelligence and know how to read, thanks to their urban education
they also mostly wear tunics instead of undershirts, to show their slightly better wealth, and will have more colorful clothes in general as a result of their access to the store, and will have a dagger instead of hunting knife because they can afford nicer things
cheesemaker is now a towner only class, pilgrims are instead wandering chefs (both capped at 2)

adds recurve bows
not unique to the town, but hunters and rangers will both get recurve bows with slightly less damage but less stamina cost and draw time

town hunter: worse crafting but can knows polearms for spear hunting for teaching the trapper, has a green tunic, both he and the trapper also have a bit of trap skill (does not do much except make escaping from handcuffs faster, but it is good for flavor)
town fisher: the same but has a blue tunic 
town carpenter: has a red tunic and better climbing skill (tall buildings) but a less well rounded skillset though still sufficient
churchling: max spawn increased to 2 and they spawn with a church key

towners can also uniquely access young adult roles, which tend to be apprentice roles for regular towner roles
- the trapper: a young hunter with a spear and mantrap for immobilizing prey for the hunter to shoot, has a red hood in case they need to deliver goodies to their granny
- the student:, a steward/merchant in training and son of a noble, good mental stats and poor physical ones, spawns with a feather and a scroll, he must protect his virginity
- the miner: the children yearn for the mines, mostly the same as the miner but starts with a lamptern and candles, since in real life children were mostly used to keep the mines well lit
- the farmhand: a young peasant who helps with menial tasks on the farm

the merchant can now order recurve bows and packs of tunics
the town is meant to be the main, if not the only, source of either

## Why It's Good For The Game
makes towners a bit more memorable
gives tunics a use
child labor is funny
churchlings live in town, right? if an acolyte doesn't join they can do crap all in the church without a key
recurve bows are cool
also allows for bows to modulate damage with damfactor down the line

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/blacksmith.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		"Aasimar"
 	)
 	outfit = /datum/outfit/job/roguetown/adventurer/blacksmith
-	isvillager = TRUE
+	isvillager = FALSE
```
[the remark this answers is not in the document]
Comment (cornercube) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/blacksmith.dm:14]: it works right now, but  I do not know what the rework is like

**Comment by Optimism333** [comment]
Looks like a pretty well put together pr, I like the idea of making town civilian roles more memorable.

The one little balance tweak I'd suggest is that a trapper (hunter apprentice) shouldn't be better with a knife than a fully fledged hunter.

**Comment by cornercube** [comment]
> Looks like a pretty interesting pr, I like the idea of making town civilian roles more memorable.
> 
> One little balance tweak I'd suggest is that a trapper (hunter apprentice) shouldn't be better with a knife than a fully fledged hunter.

hunters and trappers now have the same knives skill
trappers also have guaranteed 2 in traps while town hunters can only get between 1 and 2
trappers are younger and more hip with new tech

**Comment by cornercube** [comment]
where did the conflicts come from?

**Comment by hermaplusplus** [comment]
> where did the conflicts come from?

Merging of #65

**Comment by Stutternov** [comment]
Re-open if ever updated to current map. No pushes in 3 weeks; tons of conflicts.

**Pull request closed without merging**